src/main.coffee

```diff
@@ -5,7 +5,15 @@ feedback = require './feedback'
 
 time = -> (new Date()).getTime()
 
-zxcvbn = (password, user_inputs = []) ->
+zxcvbn = (password, options = {}) ->
+  if options instanceof Array
+    user_inputs = options # backward-compatibility
+  else if typeof options === 'object'
```

Comment:
@paulballesty I don't know CoffeeScript well but strict equality checks using "===" is  not valid syntax in CoffeeScript my local build failed using your PR. CoffeeScript converts all loose checks into strict equality checks by default so using "===" is not needed here.